esp8266-fastled-webserver/include/configs/product/fibonacci32.h

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
SIMPLE FILE REVIEW

Each file here defines the options that can vary between the different
products.  See the file `template.h` for documentation of each of the
symbols.